File: instalacao-ci/application/models/Entity/Membro.java
package br.edu.unipampa.app.domain;

import com.fasterxml.jackson.annotation.JsonIgnore;

import javax.persistence.*;
import java.io.Serializable;
import java.util.List;

/**
 * Created by Esther Favero on 15/05/2018.
 */
@Entity(name = "membros")
public class Membro implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nome;

    @JsonIgnore
    @ManyToMany(cascade = CascadeType.PERSIST, fetch = FetchType.LAZY)
    @JoinTable(
            name = "reunioes_membros",
            joinColumns = @JoinColumn(name = "membros_id"),
            inverseJoinColumns = @JoinColumn(name = "reunioes_id")
    )
    private List<Reuniao> reunioes;

    @Column(unique = true)
    private String token;

    public Long getId() {
        return id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        if (nome == null || nome == "") {
            throw new IllegalArgumentException();
        } else {
            this.nome = nome;
        }
    }

    public List<Reuniao> getReunioes() {
        return reunioes;
    }

    public void setReunioes(List<Reuniao> reunioes) {
        this.reunioes = reunioes;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

}

File: instalacao-ci/application/models/Entity/Membro.php
<?php

namespace Entity;

use Doctrine\Common\Collections\ArrayCollection;


/**
 * @Entity
 * @Table(name = "membros")
 */
class Membro
{

    /**
     * @Id
     * @Column(type="integer", nullable=false)
     * @GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @Column(type="string", nullable=false)
     */
    private $nome;

    /**
     * @ManyToMany(targetEntity="Reuniao", mappedBy="membros")
     */
    private $reunioes;

    /**
     * @Column(type="string", unique=true)
     */
    private $token;

    /**
     * Membro constructor.
     */
    public function __construct()
    {
        $this->reunioes = new ArrayCollection();
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getNome()
    {
        return $this->nome;
    }

    /**
     * @param mixed $nome
     */
    public function setNome($nome)
    {
        $this->nome = $nome;
    }

    /**
     * @return mixed
     */
    public function getReunioes()
    {
        return $this->reunioes;
    }

    /**
     * @param mixed $reunioes
     */
    public function setReunioes($reunioes)
    {
        $this->reunioes = $reunioes;
    }

    /**
     * @return mixed
     */
    public function getToken()
    {
        return $this->token;
    }

    /**
     * @param mixed $token
     */
    public function setToken($token)
    {
        $this->token = $token;
    }

}
